requests/models.py

```diff
@@ -849,12 +849,16 @@ def raise_for_status(self):
         """Raises stored :class:`HTTPError`, if one occurred."""
 
         http_error_msg = ''
+        if isinstance(self.reason, bytes):
+            reason = self.reason.decode('utf-8')
```

Comment:
Can you change this line to decode with the error handling mode `replace`, please?